src/components/Album.js

```diff
@@ -30,82 +35,332 @@ import AddToWishlistButton from './layout/AddToWishlistButton/AddToWishlistButto
-      </Button>
-      <Tooltip title={album.albumName} arrow>
-        <AlbumTitle>{album.albumName}</AlbumTitle>
-      </Tooltip>
-      <p className="album-artist">{album.artistName}</p>
-      <div className="button-container">
-        <AddToCartButton album={album} />
-        <Button
-          variant="contained"
-          color="secondary"
-          className="wishlist-button"
-        >
-          Wishlist
+      <Box
+        className="album-item"
+        sx={{
+          height: '100%',
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'space-between',
+        }}
+      >
+        <Button onClick={handleOpenDialog} className="album-image-button">
+          <AlbumImage src={album.image} alt={album.albumName} />
         </Button>
-      </div>
-      <Grid item xs={12} sm={6} md={4}>
-        <div className="album-item">
-          {/* Image as a Button */}
-          <Button onClick={handleOpenDialog} className="album-image-button">
-            <img
-              src={album.image}
-              alt={album.albumName}
-              className="album-image"
-            />
-          </Button>
-          <p className="album-title">{album.albumName}</p>
-          <p className="album-artist">{album.artistName}</p>
-          <div className="button-container">
-            {/* <Button variant="contained" color="primary" className="buy-button">
-            Buy
-          </Button> */}
-            {/*AddToCartButton component */}
-            <AddToCartButton album={album} />
-
-            {/* <Button
+        <Tooltip title={album.albumName} arrow>
+          <AlbumTitle className="album-title">{album.albumName}</AlbumTitle>
+        </Tooltip>
+        <Typography className="album-artist">{album.artistName}</Typography>
+        <Box className="button-container">
+          <AddToCartButton album={album} />
+          <Button
             variant="contained"
             color="secondary"
             className="wishlist-button"
           >
             Wishlist
-          </Button> */}
-            {/*AddToCartButton component */}
-            <AddToWishlistButton album={album} />
-          </div>
-        </div>
-
-        {/* Dialog */}
-        <Dialog open={openDialog} onClose={handleCloseDialog}>
-          <DialogTitle>{album.albumName}</DialogTitle>
+          </Button>
+        </Box>
+        <Dialog
+          open={openDialog}
+          onClose={handleCloseDialog}
+          className="album-modal-main"
+        >
+          <DialogTitle
+            sx={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              alignItems: 'center',
+            }}
+          >
+            {album.albumName}
+            <Box sx={{ cursor: 'pointer' }} onClick={handleCloseDialog}>
+              <CloseIcon />
+            </Box>
+          </DialogTitle>
           <DialogContent>
             <DialogContentText>
```

Comment:
I'm seeing a bunch of these errors:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/a9681a74-2398-4fd8-845c-986cb1703c6a)

This is occuring because `<DialogContentText>` ends up being rendered as a `<p>` tag; and proper HTML does not allow nesting block tags inside a `<p>`. We can avoid this by using a regular `<div>` here instead:

```suggestion
            <div>
```